### PR TITLE
Fixes #12132: Missing depedency to use rudder-slapd on rpm systems

### PR DESCRIPTION
**Pull request by VinceMacBuche**
https://www.rudder-project.org/redmine/issues/12132

### CODE REVIEW

File: rudder-inventory-ldap/SPECS/rudder-inventory-ldap.spec [hunk 1]
```diff
@@ -84,19 +84,13 @@ Requires: rsyslog openssl
 #Specific requirements
 
 # Reference for suse_version : https://en.opensuse.org/openSUSE:Build_Service_cross_distribution_howto
-%if 0%{?suse_version} && 0%{?suse_version} < 1100
```
Comment (amousset) [inline rudder-inventory-ldap/SPECS/rudder-inventory-ldap.spec]: It's not the same require

Comment (VinceMacBuche) [reply rudder-inventory-ldap/SPECS/rudder-inventory-ldap.spec]: We are not building for < SLES11

Comment (VinceMacBuche) [reply rudder-inventory-ldap/SPECS/rudder-inventory-ldap.spec]: This line is useless

**Comment by VinceMacBuche** [comment]
OK, merging this PR

**Pull request merged**